src/Tribe/Tickets.php

```diff
@@ -642,13 +642,34 @@ public function attendee_decreases_inventory( array $attendee ) {
 			return true;
 		}
 
+		/**
+		 * Get All Attendees by ticket/attendee ID
+		 *
+		 * @since TBD
+		 *
+		 * @param int $attendee_id
+		 * @return array
+		 */
+		public function get_all_attendees_by_attendee_id( $attendee_id ) {
+			$attendees_query = new WP_Query( array(
+				'p'         => $attendee_id,
```

Comment:
Do we have some sort of "rule" on where to sanitize data when we make queries? Perhaps, taking into consideration that is a public method, we may want to sanitize the `$attendee_id` and be sure that is an int

Reply:
Not sure we have a rule, but I think it is a good idea.